src/app/models/field/util/textValidationOptionsSchema.ts

```diff
@@ -0,0 +1,39 @@
+import { Schema } from 'mongoose'
```

Comment:
this isn't a util since it contains a schema definition. perhaps `common` might be better for the folder name?